doc/decisions/0007-feature-toggles-development.md

```diff
@@ -0,0 +1,92 @@
+# 0007. Feature toggles development
+
+| Date | Tags |
+|---|---|
+| 2018-05-25 | development |
+
+
+## Status
+
+Proposed
+
+## Context
+
+This change describes a methodology of development which we believe will help
+us increase the confidence and speed in the development process.
+
+### What problem are we trying to solve
+
+Currently the time between a PR is sent and the code gets merged is between 1 day
+to in some case 4 to 5 days.
```

Comment:
I agree with your main point whenever it is possible to isolate the change and hide it behind a flag - no arguments here. But can we attack this problem from the other direction as well? What can we do to reduce the merge time of a PR?

Reply:
All the usual suspects, we can always decrease by trying to speed up manual QA ( I feel clojure review time is generally faster, as it a simple task), having more tests so that manual qa needs to check less. Hire more, smaller PRs etc. But ultimately I do personally think manual QA does not scale well, not questioning the need of it, just how much we want to rely on it.

Reply:
Can use this https://github.com/status-im/status-react/blob/develop/scripts/pr-statistics.sh :)